packages/solicitudes/src/use-cases/db/asignaturas/delete.asignatura.use-cases.js

```diff
@@ -1,12 +1,8 @@
 const { Logger } = require('@siiges-services/shared');
-const { checkers } = require('@siiges-services/shared');
 
-const deleteAsignatura = (findOneAsignaturaQuery, deleteAsignaturaQuery) => async (
+const deleteAsignatura = (deleteAsignaturaQuery) => async (
   identifierObj,
 ) => {
-  const asignatura2 = await findOneAsignaturaQuery(identifierObj);
-  checkers.throwErrorIfDataIsFalsy(asignatura2, 'asignaturas', identifierObj.id);
-
   Logger.info('[asignatura/delete]: Deleting asignatura');
```

Comment:
change for something like this

const asignatura = await findOneAsignaturaQuery(identifierObj);
  checkers.throwErrorIfDataIsFalsy(asignatura, 'asignaturas', identifierObj.id);

  const asignaturaDeleted = await deleteAsignaturaQuery(identifierObj);
  return diligenciaDeleted;